### ISSUE TITLE
[Proposal] Add inline-level components, such as <span>

### ISSUE THREAD

**Issue by Hanks10100** [alibaba/weex#1740]
We don't have any inline level components yet, the `<text>` component is block level by default, indeed, it will compiled to `<div>` on the web. We can't achieve to write two words in one line with different colors or font style.

## Proposal

I think we should support to use `<span>` component. To simplify the implementation, we can add a convention that **`<span>` must be the child of `<text>`**, as well as other inline-level components.

## Example

```html
<div>
    <text>
        <span style="color:red">R</span>
        <span style="color:green">G</span>
        <span style="color:blue">B</span>
    </text>
</div>
```

This demo should render the string `RGB` with three different colors.

## What's More

+ `<em>`, `<i>`, `<b>`, `<strong>`, `<small>`, could also be supported. (maybe it's not necessary)
+ If we support `display: inline-block;`, It would be much easier to achieve rich text (#835).
+ It's possible to create a `<markdown>` component without writing native codes.

**Comment by Jinjiang** [comment]
I think it's more better to encapsulated by a frontend custom component or functional Vue component instead based on `<richtext>`. They don't make more native feature and a frontend-only solution is enough.

**Comment by Jinjiang** [comment]
The discussion was merged into #835
